tests/lib/rules/named-functions-in-promises.js

```diff
@@ -39,6 +39,30 @@ eslintTester.run('named-functions-in-promises', rule, {
     }, {
       code: 'user.save().finally(_finallyDo);',
       parserOptions: { ecmaVersion: 6 },
+    }, {
+      code: 'user.save().then(() => this._reloadUser(user));',
+      parserOptions: { ecmaVersion: 6 },
+      options: [{
+        allowConciseArrow: true,
+      }],
+    }, {
+      code: 'user.save().catch(err => this._handleError(err));',
+      parserOptions: { ecmaVersion: 6 },
+      options: [{
+        allowConciseArrow: true,
+      }],
+    }, {
+      code: 'user.save().finally(() => this._finallyDo());',
+      parserOptions: { ecmaVersion: 6 },
+      options: [{
+        allowConciseArrow: true,
+      }],
+    }, {
+      code: 'user.save().then(() => user.reload());',
```

Comment:
I'm not sure if I agree with this case. I think you should probably write `user => this._reloadUser(user)` anyway, to be able to test it nicely. What do you think @bardzusny? I see it was like this before but then it was updated..